.github/workflows/release-please.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   release-please:
     runs-on: ubuntu-latest
     steps:
-      - uses: google-github-actions/release-please-action@b1f383133aa4cc90eca1d35ae7ac7d96c1e83d72
+      - uses: google-github-actions/release-please-action@v3
```

Comment:
it would be awesome if this works out of the box! I wonder if there are now more configuration knobs we can tweak to support our non-standard versioning strategies and stop having to tweak release-please-config.json so much

Reply:
Afaict from reading through all the manifest and configs, our non-standard versioning where we override the conventional versioning with release-as is the accepted way of doing this. But i'll double check later too.

Reply:
https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#configfile 

```
    "path/to/myPyPkgA": {
      // when a default release-as is set, this is how you revert to using
      // conventional commits version bumping for an individual package.
      // Note: if present and set to "", the behavior will always be to use
      // conventional commits, regardless of presence or absence of default
      // release-as config.
      "release-as": "",
      "package-name": "coolio-pkg",
      // our change log is located at path/to/myPyPkgA/docs/CHANGES.rst
      "changelog-path": "docs/CHANGES.rst"
    },
    ```

Reply:
yeah, for sure. but I remember them telling me that now that they've refactored it a bit, we can somehow plug in and setup a custom strategy like "for packageA, only ever bump the `patch` version by default"